File: semester-5/scripting-langs/lab6/Task3.java
import java.io.File;
import java.io.FileNotFoundException;
import java.util.Scanner; 

class Task3 {
	public static void main(String[] args) {
		try {
			System.out.println("Country: ");
			Scanner stdIn = new Scanner(System.in);
			String country = stdIn.nextLine();

			File covidFile = new File("Covid.txt");
			Scanner fileReader = new Scanner(covidFile);
			int totalCases = 0;

			fileReader.nextLine();
			while (fileReader.hasNextLine()) {
				String[] row = fileReader.nextLine().split("\t");
		
				if (row[6].equals(country)) {
					totalCases += Integer.parseInt(row[4]);
				}
			}

			fileReader.close();
			stdIn.close();

			System.out.println(totalCases);
		} catch (FileNotFoundException e) {
			System.out.println("An error occurred.");
			e.printStackTrace();
		}
	}
}

File: semester-5/scripting-langs/lab6/task3.py
import time

FILE_PATH = 'Covid.txt'

def calc_cases(file_name, country):
    with open(file_name, 'r') as covid_file:
        covid_file.readline() # skip header

        cases = 0
        for line in covid_file.readlines():
            row = line.split('\t')
            if row[6] == country:
                try:
                    cases += int(row[4])
                except:
                    pass

        return cases

country = input("Country: ")

try:
    print(calc_cases(FILE_PATH, country))
except:
    print('No Covid.txt file')